migrate_bpm.py

```
@@ -0,0 +1,215 @@
+    extraction_failed_count = 0
+
+    # 各楽曲を処理
+    for idx, song in enumerate(songs, 1):
+        print(f"[{idx}/{total_songs}] {song.song_id}")
+        print(f"   source_dir: {song.source_dir}")
+        
+        # ファイルパスを構築
+        file_path = find_audio_file(base_path, song.source_dir, song.song_id)
+        
+        if file_path is None:
+            print(f"   ❌ ファイルが見つかりません")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)